packages/address-validator/package.json

```diff
@@ -20,6 +20,6 @@
     },
     "devDependencies": {
         "jest-environment-jsdom": "29.7.0",
-        "rimraf": "^5.0.5"
+        "rimraf": "^6.0.1"
```

Comment:
todo later: maybe packages that are already defined in root package json shoudln't be present in particular packages package.jsons? cc @Nodonisko

Reply:
They shouldn't.

Reply:
ok, @karliatto please just rebase and let's merge it now. Removing root defined packages should be done as a followup.